setup/src/Magento/Setup/Test/Unit/Module/Di/Code/Reader/FileScannerTest.php

```diff
@@ -1,40 +0,0 @@
-<?php
-/**
- * Copyright © Magento, Inc. All rights reserved.
- * See COPYING.txt for license details.
- */
-declare(strict_types=1);
-
-namespace Magento\Setup\Test\Unit\Module\Di\Code\Reader;
-
-use Magento\Setup\Module\Di\Code\Reader\FileScanner;
-use PHPUnit\Framework\TestCase;
-
-class FileScannerTest extends TestCase
-{
-    /**
-     * @var FileScanner
-     */
-    private $object;
-
-    protected function setUp(): void
-    {
-        $this->object = new FileScanner(
-            __DIR__ . DIRECTORY_SEPARATOR . '_files' . DIRECTORY_SEPARATOR . 'classes.php'
-        );
-    }
-
-    public function testGetClassesReturnsAllClassesAndInterfacesDeclaredInFile()
```

Comment:
Why this test was removed?

Reply:
Hi @ihor-sviziev , this test is checking a logic that was removed. getClasses() method is no longer exists and I got rid of it.